scripts/tools/common.yml

```diff
@@ -180,7 +200,7 @@ members:
       desc: "[out] 32-bit unsigned-integer"
     - type: uint64_t
       name: "ui64"
-      desc: "[out] 32-bit unsigned-integer"
+      desc: "[out] 64-bit unsigned-integer"
```

Comment:
this correction is unrelated to this PR, I think would be good to make this correction as separate PR anyway since we can merge that quickly while we revise the proposal here?

Reply:
ok

Reply:
done